src/components/r.vue

```diff
@@ -0,0 +1,759 @@
+<template v-if="semesters">
```

Comment:
What is this? It looks like a leftover of merge conflicts.

Reply:
Yes it was a leftover merge conflict! Deleted r.vue